container/util.py

```diff
@@ -57,7 +57,11 @@ def filter_image(
             oci_client=oci_client,
         )
 
-    manifest = oci_client.manifest(image_reference=source_ref)
+    try:
+        manifest = oci_client.manifest(image_reference=source_ref)
+    except NotImplementedError as e:
+        logger.warning(f'could not filter image {e}. Skipping...')
```

Comment:
do not (almost) silently skip. let the exception propagate. it should be up to the caller do decide whether or not to ignore this